src/util.h

```diff
@@ -9,16 +9,26 @@
 
 #include "../include/secp256k1.h"
 
-#include <stdlib.h>
-#include <stdint.h>
-#include <stdio.h>
 #include <limits.h>
+#include <stddef.h>
+#include <stdint.h>
+#if SECP256K1_HAVE_STDIO_H
+#  include <stdio.h>
+#else
+#  pragma message "<stdio.h> not available, disabling debugging output for fatal library errors."
```

Comment:
This one should only be displayed when the default callbacks are active.

(Or I change my mind and just get rid of it entirely... I'm not sure if these notes are too annoying/intrusive if it's too hard to silence them.)